.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,6 +21,7 @@ A clear and concise description of what you expected to happen.
 Attach logs from the following directory:
 For Windows: %TEMP%/Bridge to Kubernetes
 For OSX/Linux: $TMPDIR/Bridge to Kubernetes
+For Visual Studio users: %temp%\Microsoft.VisualStudio.Kubernetes.Debugging
```

Comment:
Please consider updating so that Visual Studio users provide both logs.
Maybe: `If you are a Visual Studio user, please also provide these logs: %temp%\Microsoft.VisualStudio.Kubernetes.Debugging`